### PR TITLE
refactor(logs): drop "Amazon Q Logs" channel and just have "Amazon Q"

### PR DESCRIPTION
**Pull request by nkomonen-amazon**
## Problem

We didn't have much use for the old "Amazon Q" channel in addition to "Amazon Q Logs". All the logs we used were in "Amazon Q Logs".

## Solution

- Now, just have the useful "Amazon Q Logs" channel.
- Move the Amazon Q Language Server logs in to Amazon Q Logs to unify all logs

---

License: I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amazonq/src/extension.ts [hunk 1]
```diff
@@ -97,10 +97,8 @@ export async function activateAmazonQCommon(context: vscode.ExtensionContext, is
     globals.manifestPaths.endpoints = context.asAbsolutePath(join('resources', 'endpoints.json'))
     globals.regionProvider = RegionProvider.fromEndpointsProvider(makeEndpointsProvider())
 
-    const qOutputChannel = vscode.window.createOutputChannel('Amazon Q', { log: true })
-    const qLogChannel = vscode.window.createOutputChannel('Amazon Q Logs', { log: true })
-    await activateLogger(context, amazonQContextPrefix, qOutputChannel, qLogChannel)
-    globals.outputChannel = qOutputChannel
+    const qLogChannel = vscode.window.createOutputChannel('Amazon Q', { log: true })
```
Comment (justinmk3) [inline packages/amazonq/src/extension.ts:100]: Let's keep "Logs" for now. That aligns with Toolkit, and also signals that these are potentially low-level messages. And it keeps the door open for potentially in the future having higher-level messages in a `Amazon Q` channel

Comment (nkomonen-amazon) [reply packages/amazonq/src/extension.ts]: Sounds good, changed

File: packages/core/src/amazonq/lsp/lspClient.ts [hunk 2]
```diff
@@ -201,6 +201,9 @@ export async function activate(extensionContext: ExtensionContext) {
             // this is used by LSP to determine index cache path, move to this folder so that when extension updates index is not deleted.
             extensionPath: path.join(fs.getUserHomeDir(), '.aws', 'amazonq', 'cache'),
         },
+        // Log to the Amazon Q Logs so everything is in a single channel
+        // TODO: Add prefix to the language server logs so it is easier to search
+        outputChannel: globals.logOutputChannel,
```
Comment (justinmk3) [inline packages/core/src/amazonq/lsp/lspClient.ts:206]: Nice!

**Review by justinmk3 — APPROVED** [review]

**Pull request merged**